packages/charts/chart-web-components/src/horizontal-bar-chart/horizontalbarchart.spec.ts

```diff
@@ -317,6 +317,21 @@ test.describe('horizontalbarchart - Basic', () => {
     await expect(page.getByText('Unmonitored')).toBeVisible();
   });
 
+  test('Should render bars and bar labels properly', async ({ page }) => {
```

Comment:
rename the spec file to match the hyphen based naming convention used in other charts.